Battlegroup.cat

```diff
@@ -20,9 +20,9 @@
         <categoryLink id="3546-6658-7a79-214d" name="Princeps Seniores" hidden="false" targetId="c130-1260-4c15-147a" primary="false">
           <modifiers>
             <modifier type="increment" field="bf78-6b8f-80d8-996a" value="1.0">
-              <conditions>
-                <condition field="selections" scope="force" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" childId="4225-27f8-9c8b-e56e" type="atLeast"/>
-              </conditions>
+              <repeats>
+                <repeat field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" childId="model" repeats="1" roundUp="false"/>
+              </repeats>
             </modifier>
           </modifiers>
           <constraints>
```

Comment:
I believe the contents of this constraints block is still limiting to 1 princeps seniores per force

Reply:
For some reason I didn't get the error, so did a test with a clean install of battlescribe and the Titanicus files and now I get the error, I will cancel the pull request till I get this code sorted